pkg/cloudprovider/aws/constraints.go

```diff
@@ -32,18 +32,16 @@ const (
 )
 
 var (
-	AWSLabelPrefix             = "node.k8s.aws/"
-	CapacityTypeLabel          = AWSLabelPrefix + "capacity-type"
-	LaunchTemplateIdLabel      = AWSLabelPrefix + "launch-template-id"
-	LaunchTemplateVersionLabel = AWSLabelPrefix + "launch-template-version"
-	SubnetNameLabel            = AWSLabelPrefix + "subnet-name"
-	SubnetTagKeyLabel          = AWSLabelPrefix + "subnet-tag-key"
-	SecurityGroupNameLabel     = AWSLabelPrefix + "security-group-name"
-	SecurityGroupTagKeyLabel   = AWSLabelPrefix + "security-group-tag-key"
-	AllowedLabels              = []string{
+	AWSLabelPrefix           = "node.k8s.aws/"
+	CapacityTypeLabel        = AWSLabelPrefix + "capacity-type"
+	LaunchTemplateIdLabel    = AWSLabelPrefix + "launch-template-id"
```

Comment:
Thoughts on moving to launch template name to avoid the order-of-operations problem when creating infrastructure?

Reply:
Good idea, will do